apps/src/sites/studio/pages/maker/setup.js

```diff
@@ -31,38 +32,151 @@ const BoardSetupStatus = React.createClass({
     return `${baseFormURL}?${userAgentFieldFill}&${setupStatesFieldFill}`;
   },
 
+  detect() {
+    this.setState({...this.getInitialState(), isDetecting: true});
+
+    if (!isChrome()) {
+      this.fail('IsChrome');
```

Comment:
It looks like there is a discrete set of allowing things that can pass fail. Should we have some sort of enum, such that we end up doing `this.fail(BoardStatus.IsChrome)`?